source/common/config/bootstrap_json.cc

```diff
@@ -28,10 +28,6 @@ void BootstrapJson::translateClusterManagerBootstrap(
     Config::CdsJson::translateCluster(*json_sds->getObject("cluster"),
                                       absl::optional<envoy::api::v2::core::ConfigSource>(),
                                       *cluster, stats_options);
-    Config::Utility::translateEdsConfig(
```

Comment:
I guess we are starting to break the config translator CLI tool at this point, but this is probably inevitable. We should aim to retire that tool at the end of the next release cycle I think and it can exist with diminished capacity until then.

Reply:
I can also reinstate this if we want to?